docs/docs/09-typescript/01-resources.md

```diff
@@ -26,7 +26,7 @@ main((root) => {
 });
 ```
 
-API documentation for the `cloud` namespace is available [here](../04-standard-library/cloud).
+API documentation for the `cloud` namespace is available [here](../04-standard-library/cloud/api.md).
```

Comment:
this is actually a resource called `cloud.Api()` (api gateway): https://www.winglang.io/docs/standard-library/cloud/api
I believe you need this page: https://www.winglang.io/docs/category/cloud

Reply:
Nice catch, thanks @tsuf239 , will make a new PR for this